pandas/tests/dtypes/test_common.py

```diff
@@ -352,11 +354,13 @@ def test_is_not_integer_dtype(dtype):
 
 @pytest.mark.parametrize(
     "dtype",
-    [pd.Series([1, 2])]
-    + SIGNED_INT_DTYPES
-    + to_numpy_dtypes(SIGNED_INT_DTYPES)
-    + SIGNED_EA_INT_DTYPES
-    + to_ea_dtypes(SIGNED_EA_INT_DTYPES),
```

Comment:
what is mypy's complaint about this?  I think the current idiom is more beginner-friendly than the args-expansion idiom

Reply:
IIRC the issue is that you can't really do list addition and have it type inference appropriately all in one pass, at least not from a static analysis perspective

Instead of doing this could you also just initialize this a line up with the appropriate type comment and just pass that variable into the parametrization?

Reply:
@WillAyd Just so I understand exactly what you're suggesting, are you talking about initializing the list of dtypes originially constructed by list addition through unpacking them into a list outside of the parametrizations, giving that list a type stub (presumably List[Dtype]), and then passing it into the paramaterization?

Reply:
No unpacking just defined this as a variable on the line above. 

```python
signed_int_dtypes = []  # type: List[Union[pd.Series, Dtype]]
```

And then just use that as the argument here.  

Side note - I'm assuming the type above works but can't remember what the mypy complaint was and what it should be just glancing at it. Might need to tweak depending on what you see

Reply:
Okay, those variables have been added and given type stubs for the tests that were throwing mypy errors. It looks like list addition is working without any problems now.